eng/targets/Settings.props

```diff
@@ -49,6 +49,7 @@
 
     <!-- VS does not require Windows PDBs to be published anymore. -->
     <PublishWindowsPdb>false</PublishWindowsPdb>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
```

Comment:
This doesn't look correct - the property should default to `false`. This property should be set to `true` only for projects with analyzers or generators, i.e. Features layer and Code Style layer projects for analyzers.